index.html

```diff
@@ -1176,7 +1201,8 @@ <h2 id="docconformance">
             </td>
             <td>
               <p>
-                If not provided an author defined accessible name by other methods: <strong class="nosupport">No `role`</strong>, and <strong>no `aria-*` attributes</strong> except
+                If not provided an author defined accessible name by other methods:
```

Comment:
reads awkwardly. maybe "If no author defined accessible name is provided by other methods:"